boards/openmote-cc2538/include/periph_conf.h

```diff
@@ -82,6 +82,13 @@
 /** @} */
 
 /**
+ * @name ADC configuration
+ * @{
+ */
+#define ADC_NUMOF           (8)
```

Comment:
It is actually the maximum number of possible ADC available on the cc2538  , but while testing I found that only 5 of the ADC i.e PA2 to PA6 could be accessed on the Openmote-cc2538 . 

For convenience I have defined the ADC pins by their labels printed on the OpenBase . If you test the code with the OpenBase you will notice that the ON/SLEEP and CTS/DI07 pins correspond to the ADC lines PA3 and PA6 respectively.  

I will change the number of ADC lines to 5 . Ty